packages/framework/src/Form/Constraints/NotNegativeMoneyAmountValidator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Form\Constraints;
+
+use Shopsys\FrameworkBundle\Component\Money\Money;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class NotNegativeMoneyAmountValidator extends ConstraintValidator
+{
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Money\Money|null $value
+     * @param \Symfony\Component\Validator\Constraint $constraint
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        if ($value instanceof Money) {
+            $zeroMoney = Money::fromInteger(0);
+
+            if ($value->isLessThan($zeroMoney)) {
```

Comment:
I guess we should implement `isPositive`, `isNegative` and `isZero` then. Maybe a static method for zero money should be also implemented (`Money::zero()`), because these use-cases are quite common.

All new methods should be unit-tested.

Reply:
Added this in bff9560fade13d1f75c80c8546f1177cbe395c24